pkg/apis/kops/v1alpha1/conversion.go

```diff
@@ -235,15 +235,25 @@ func Convert_kops_EtcdMemberSpec_To_v1alpha1_EtcdMemberSpec(in *kops.EtcdMemberS
 }
 
 func Convert_v1alpha1_InstanceGroupSpec_To_kops_InstanceGroupSpec(in *InstanceGroupSpec, out *kops.InstanceGroupSpec, s conversion.Scope) error {
+	err := autoConvert_v1alpha1_InstanceGroupSpec_To_kops_InstanceGroupSpec(in, out, s)
```

Comment:
Changed order here to preserve the logic, now that the autogenerated code has Zones fields...

Reply:
Will this impact auto-generated code?

Reply:
Should not.